README.md

```diff
@@ -13,67 +12,52 @@ Please check out the following if you are new to Twilio's Programmable Voice or
 
 ```sh
 yarn add @twilio/voice-react-native-sdk
 ```
 
-## Getting Started
+Once the package has been installed to your React Native application, there are further steps that you will need to take for both iOS and Android platforms. Please see the supporting documentation below.
 
-Please see this document on getting set up for your target platform: [Getting Started](docs/getting-started.md).
+## Supporting Documentation
 
-## Usage
+### Getting Started
 
-The following example demonstrates how to make and receive calls. You will need to implement your own `getAccessToken()` method for it to work properly. Please see [Access Tokens](#access-tokens) section for more details or check out the [iOS](https://github.com/twilio/voice-quickstart-ios) and [Android](https://github.com/twilio/voice-quickstart-android) quickstart for examples on how to generate the tokens.
-For more information on the Voice React Native SDK API, refer to the [API Docs](https://github.com/twilio/twilio-voice-react-native/blob/latest/docs/api/voice-react-native-sdk.md) or see our [Reference App](https://github.com/twilio/twilio-voice-react-native-app).
+#### iOS
+Learn how to get started for the [iOS platform](/docs/getting-started-ios.md).
 
-```ts
-import { Voice } from '@twilio/voice-react-native-sdk';
+#### Android
+Learn how to get started for the Android platform if you are using [Java](/docs/getting-started-android-java.md) or [Kotlin](/docs/getting-started-android-kotlin.md).
 
-const token = getAccessToken();
-const voice = new Voice();
+### Migration Guide
+If you are migrating from a version of the Twilio Voice React Native SDK `< 1.0.0.beta.4` to a version `>= 1.0.0.beta.4`, please see [this](/docs/migration-guide-beta.4.md) document.
 
-// Allow incoming calls
-await voice.register(token);
+### Customizing Notifications
+To customize the appearance and content of your application's notifications, please see [this](/docs/customize-notifications.md) document.
 
-// Handle incoming calls
-voice.on('callInvite', (callInvite) => {
-  callInvite.accept();
-});
+### Outgoing Call Ringback Tone
+To enable your application to play a ringback tone while making an outgoing call, please see [this](/docs/play-outgoing-call-ringback-tone.md) document.
 
-// Make an outgoing call
-const call = await voice.connect(token, params);
-```
+### Out-of-band PushKit Handling
+To have your application implement or use its own `PushKit` delegate module, please see [this](/docs/applications-own-pushkit-handler.md) document.
 
-### Ringtone
+### Out-of-band Firebase Messaging Service
+To have your application implement or use a different `FirebaseMessagingService` (such as OneSignal or RNFirebase), please see [this](/docs/out-of-band-firebase-messaging-service.md) document.
 
-To use a custom ringtone for outgoing calls, please see the documentation located at [docs/play-outgoing-call-ringback-tone.md](./docs/play-outgoing-call-ringback-tone.md).
+## Issues and Support
```

Comment:
I see "Expo is not supported" is in the `common issues`, but should it be moved here (to the main README) since it is a main question from many customers?

Reply:
That's a good question. I would like to keep the common issues in one place, so I'd prefer not to. But I can see the value. WDYT @charliesantos?

Reply:
Let's keep it in the common issues. Also, we should pin the github issue where some customers were able to set it up properly.

Reply:
That issue is pinned in our Github Issues.